core/src/main/scala/org/apache/spark/deploy/master/MasterArguments.scala

```diff
@@ -68,6 +73,14 @@ private[master] class MasterArguments(args: Array[String], conf: SparkConf) {
       webUiPort = value
       parse(tail)
 
+    case ("--rpc-submission-port") :: IntParam(value) :: tail =>
```

Comment:
I'd rather not add command line parameters. Just keep everything in the conf file.